apps/docs/src/docs/components/demo/PopoverStyles.vue

```diff
@@ -0,0 +1,20 @@
+<template>
+  <!-- #region template -->
+  <div class="d-flex gap-2">
+    <BButton id="popover-style">Button</BButton>
+
+    <BPopover
+      :model-value="true"
```

Comment:
```suggestion
      show
```
true doesn't work here. It need to be variable.

Reply:
@xvaara I don't think the model-value passed in for something that uses vue's defineModel has to be a variable. I couldn't tell one way or the other from the docs, but the seems conclusive: https://play.vuejs.org/#eNqdUstOwzAQ/BXLJ5BKc4BTlEYC1ANIPNSWmy8h2QQXx478CJWi/Dtrh4a0VD1wSrwzO55dT0dvm2beOqAxTUyueWOJAeualEleN0pb0mkoe1JqVRNGkcnoCK3xK+BJFSD2hHk0KXphT2cyV9JYUgfmgqDihdUOLplMouFWvA8PFrA3s4AnQpKCt+mbLKDkEoqhOSbJRD9Kk8iTRvYGRU8RSRyKV20mHCwY9Zcz+qd9BeV/upNo4pvOqDU4bsmr+dYoiYvt/AWM5qpuuAD90liO62A0JgHxWCaE+noMNa8+29fzD8g/T9S3ZudrjL5qMKBb9DNiNtMV2AFerp9hh/8jiJM4gewz4AqMEs57HGh3ThZoe8ILbh9CBrisNma5syDNfqiwHmT2gR8yc39m9F+71/Ob0Mdkj1s8ytFxQInIZOUfA9Vw7YcBGzITepN3pQRkMr3osJo5YWNSZsJAfz59Xfej1qOfwxfuvwExohlC 

I think the naked `show` prop provides a better dev experience, but there are a bunch of places in the docs where I've been using `:model-value="true"` pattern (`BAlert`s scattered through the docs for one) - I think I'll eventually convert them to `show` but it's a higher priority if they're actually broken. Also, I may update the migration guide if both techniques work, there may be cases where the `:model-value="true"` makes more sense.